2021-04-24 09:50:25,335 INFO    MainThread:15651 [wandb_setup.py:_flush():69] setting env: {}
2021-04-24 09:50:25,335 INFO    MainThread:15651 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-24 09:50:25,336 INFO    MainThread:15651 [wandb_init.py:_log_setup():336] Logging user logs to /home/arora/kaggle/shopee/src/wandb/run-20210424_095025-hw9jrudz/logs/debug.log
2021-04-24 09:50:25,336 INFO    MainThread:15651 [wandb_init.py:_log_setup():337] Logging internal logs to /home/arora/kaggle/shopee/src/wandb/run-20210424_095025-hw9jrudz/logs/debug-internal.log
2021-04-24 09:50:25,336 INFO    MainThread:15651 [wandb_init.py:init():369] calling init triggers
2021-04-24 09:50:25,336 INFO    MainThread:15651 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-24 09:50:25,336 INFO    MainThread:15651 [wandb_init.py:init():418] starting backend
2021-04-24 09:50:25,336 INFO    MainThread:15651 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-24 09:50:25,346 INFO    MainThread:15651 [backend.py:ensure_launched():123] starting backend process...
2021-04-24 09:50:25,353 INFO    MainThread:15651 [backend.py:ensure_launched():128] started backend process with pid: 15708
2021-04-24 09:50:25,354 INFO    MainThread:15651 [wandb_init.py:init():423] backend started and connected
2021-04-24 09:50:25,355 INFO    MainThread:15651 [wandb_init.py:init():465] updated telemetry
2021-04-24 09:50:25,356 INFO    MainThread:15651 [wandb_init.py:init():484] communicating current version
2021-04-24 09:50:25,890 INFO    MainThread:15651 [wandb_init.py:init():489] got version response 
2021-04-24 09:50:25,890 INFO    MainThread:15651 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-24 09:50:25,945 INFO    MainThread:15651 [wandb_init.py:init():522] starting run threads in backend
2021-04-24 09:50:26,070 INFO    MainThread:15651 [wandb_run.py:_console_start():1524] atexit reg
2021-04-24 09:50:26,070 INFO    MainThread:15651 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-24 09:50:26,071 INFO    MainThread:15651 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-24 09:50:26,072 INFO    MainThread:15651 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-24 09:50:26,072 INFO    MainThread:15651 [wandb_init.py:init():546] run started, returning control to user process
2021-04-24 09:50:40,071 INFO    MainThread:15651 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-24 09:50:40,072 INFO    MainThread:15651 [wandb_run.py:_restore():1466] restore
2021-04-24 09:50:42,131 INFO    MainThread:15651 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 546
  total_bytes: 546
}

2021-04-24 09:50:42,952 INFO    MainThread:15651 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 546
  total_bytes: 546
}

2021-04-24 09:50:43,054 INFO    MainThread:15651 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4979
  total_bytes: 4979
}

2021-04-24 09:50:43,156 INFO    MainThread:15651 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4979
  total_bytes: 4979
}

2021-04-24 09:50:43,257 INFO    MainThread:15651 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4979
  total_bytes: 4979
}

2021-04-24 09:50:43,359 INFO    MainThread:15651 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4979
  total_bytes: 4979
}

2021-04-24 09:50:43,461 INFO    MainThread:15651 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4979
  total_bytes: 4979
}

2021-04-24 09:50:44,596 INFO    MainThread:15651 [wandb_run.py:_show_files():1838] logging synced files
